README.md

```diff
@@ -67,10 +67,32 @@ kubectl delete namespace rbac-manager
 
 RBAC Manager can also be run as a controler using custom resources to store this format of RBAC configuration. These custom resources are `rbacdefinitions`. The RBAC Manager controller listens for `rbacdefinition` updates, and will automatically make the requested changes when a `rbacdefinition` is created or updated.
 
-Sample Kubernetes configuration for this pattern is available in `example/k8s/controller`. You can run this example in your cluster with this command:
+You can deploy the controller using helm:
```

Comment:
I agree that we should add Helm here as the default example, but I think it may still be worth referencing the `example/k8s/controller` directory for people that don't want to use Helm. Maybe that could be added after the Helm example?

Reply:
I generated a dry-run of all the manifests from helm and put it in examples/k8s. It has a nice "don't' edit me" header. For now we can manually regenerate that I think?